src/schedule/Schedule.ts

```diff
@@ -87,10 +92,11 @@ export class Schedule extends LogEmitter {
    * You might not want to await the promise if your specified delay is big.
    *
    * @param name the job to run
+   * @param parameters the parameters that will be passed to the job for this single run; passing nothing will run the job with no parameters, NOT the parameters used for the scheduled runs
```

Comment:
Just for clarification: with "NOT the parameters used for the scheduled runs" you want to say that they are not used as fallback in case no parameter would be provided?
Maybe sth like `passing nothing will run the job with no parameters, regardless of potentially set job parameters for the scheduled runs`?

Reply:
Exactly. Yeah, that sounds better!